clusters/app.ci/registry-access/baremetal-qe/baremetal-qe.yaml

```
@@ -0,0 +1,63 @@
+# this is the Namespace in which your ServiceAccount will live
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Automation ServiceAccounts for BareMetal QE
+    openshift.io/display-name: BareMetal QE CI
+  name: baremetal-qe
+---
+# this is the ServiceAccount whose credentials you will use
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: image-puller
+  namespace: baremetal-qe
+---
+# this grants your ServiceAccount rights to pull images
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: baremetal-qe-image-puller-binding
+  # the namespace from which you will pull images
+  namespace: ocp
```

Reopening https://github.com/openshift/release/pull/55244/files#r1705654247
```suggestion
  namespace: ci
```